README.md

```diff
@@ -1 +1,4 @@
-# hw01
\ No newline at end of file
+# hw01
+# **Yanyi Leng** 
+Hi there! My name is **Yanyi Leng**. I pursued Bachelor of Arts in Psychology at *Univeristy of Southern California*, and now I am pursuing my Master degree at *The University of Chicago*. Here is my Linkedin link: https://www.linkedin.com/in/yanyi-tracy-leng-146b55132/
```

Comment:
Generally we don't like to print the full URL as written text because it is long and ugly. You can use an alternative syntax to embed it in text:

```
[Here is my Linkedin link](https://www.linkedin.com/in/yanyi-tracy-leng-146b55132/)
```

[Here is my Linkedin link](https://www.linkedin.com/in/yanyi-tracy-leng-146b55132/)